tocs/4.FAQ/FAQ.md

```diff
@@ -15,6 +15,8 @@
 - [How to update the application on X261?](#how-to-update-the-application-on-x261)
 - [How to check the verion on EXMU-X261?](#how-to-check-the-verion-on-exmu-x261)
 - [Aborted to run dpu-sc?](#aborted-to-run-dpu-sc)
+- [At first boot into system, what's the account name? password?](#at-first-boot-into-system-whats-the-account-name-password)
```

Comment:
In the first boot, what is the login account and password?

Reply:
How about "For first time boot, what is the login account and password?"